CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# v1.36.0 (2025-04-07)
+
+## Release Highlights
+* Switch to igzip (x86_64) or pigz with zlib-ng (aarch64) to decompress container images ([bottlerocket-core-kit#443](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/443))
+* Add support for more AWS regions in schnauzer and host-ctr ([bottlerocket-core-kit#454](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/454))
```

Comment:
nit: this could have been just "Add support for more AWS regions", since `schnauzer` was removed